_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3146,19 +3116,7 @@ public static function update_service_api_key( $request ) {
 	 * }
 	 */
 	public static function delete_service_api_key( $request ) {
-		$service = self::validate_service_api_service( $request['service'] );
-		if ( ! $service ) {
-			return self::service_api_invalid_service_response();
-		}
-		$option = self::key_for_api_service( $service );
-		Jetpack_Options::delete_option( $option );
-		$message = esc_html__( 'API key deleted successfully.', 'jetpack' );
-		return array(
-			'code'            => 'success',
-			'service'         => $service,
-			'service_api_key' => Jetpack_Options::get_option( $option, '' ),
-			'message'         => $message,
-		);
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::delete_service_api_key( $request );
 	}
```

Comment:
Same as https://github.com/Automattic/jetpack/pull/10742/files#r237570229, but for `WPCOM_REST_API_V2_Endpoint_Service_API_Keys::delete_service_api_key` on [R450](https://github.com/Automattic/jetpack/pull/10742/files#diff-58180905242554a749dbb58ba9464cc9R450)?